src/reducers/summit_speakers/summit-speakers-list-reducer.js

```diff
@@ -56,31 +58,74 @@ const summitSpeakersListReducer = (state = DEFAULT_STATE, action) => {
             return DEFAULT_STATE;
         }
         case REQUEST_SPEAKERS_BY_SUMMIT: {
-            let { order, orderDir, term, page, perPage, ...rest } = payload;
-            return { ...state, order, orderDir, term, currentPage: page, perPage, ...rest }
+            let { order, orderDir, page, ...rest} = payload;
+
+            if (order !== state.order || orderDir !== state.orderDir || page !== state.currentPage) {
+                // if the change was in page or order, keep selection
+                return {
+                    ...state,
+                    order,
+                    orderDir,
+                    currentPage: page,
+                    ...rest
+                }
+            }
+
+            return {
+                ...state,
+                order,
+                orderDir,
+                currentPage: page,
+                selectedItems: [],
+                excludedItems: [],
+                selectedCount: 0,
+                selectedAll: false,
```

Comment:
i do think that here we need to    items: state.items.map(it => ({...it, checked: false})), to avoid a UX issue